### PR TITLE
Fixes #4 - updated stylelint and stylelint-config-standard

### LINKED ISSUE #4: add linter
- [x] add prettier
- [x] add rules same as in webcompat.com
- [x] add task to run linter

### PR DESCRIPTION
**Pull request by magsout**
Updated stylelint and stylelint-config-standard.

@madeleine-neumann could you test if it fixed this https://github.com/webcompat/design/issues/4#issuecomment-344632386 and https://github.com/webcompat/design/issues/4#issuecomment-345385364 ?

### CODE REVIEW

**Comment by magsout** [comment]
@madeleine-neumann my last commit should fixed your issue https://github.com/webcompat/design/issues/4#issuecomment-345434774

File: docs/.stylelintrc [hunk 1]
```diff
@@ -8,8 +8,6 @@
         "color-hex-length": "short",
         "color-named": "never",
         "selector-no-qualifying-type":  [ true, {"ignore": "attribute"} ],
-        "selector-no-id": true,
```
Comment (kotzendekrabbe) [inline docs/.stylelintrc:11]: please use "selector-max-i"': 0 instead of removing it

File: docs/.stylelintrc [hunk 2]
```diff
@@ -8,8 +8,6 @@
...
-        "selector-no-attribute": true,
```
Comment (kotzendekrabbe) [inline docs/.stylelintrc:12]: "selector-max-attribute": 0

File: docs/.stylelintrc [hunk 3]
```diff
@@ -42,8 +39,6 @@
         "selector-pseudo-element-colon-notation": "double",
         "selector-pseudo-class-parentheses-space-inside": "never",
         "selector-no-vendor-prefix": true,
-        "selector-no-universal": true,
```
Comment (kotzendekrabbe) [inline docs/.stylelintrc:45]: "selector-max-universal": 0

File: docs/.stylelintrc [hunk 4]
```diff
@@ -42,8 +39,6 @@
...
-        "selector-no-type": true,
```
Comment (kotzendekrabbe) [inline docs/.stylelintrc:46]: "selector-max-type": 0

**Review by kotzendekrabbe — COMMENTED** [review]
can you please use max instead of deleting the deprecated rules? thank you :)

**Comment by magsout** [comment]
@kotzendekrabbe done

**Review by kotzendekrabbe — APPROVED** [review]

**Pull request merged**